test/utils/TestKeyManager.sol

```diff
@@ -101,6 +100,10 @@ library TestKeyManager {
     }
 
     function toKeyHash(TestKey memory key) internal pure returns (bytes32) {
+        /// If the key is the root user return a hash of 0 to signify the root user.
+        if (key.keyType == KeyType.Secp256k1 && abi.decode(key.publicKey, (address)) == signerTestKeyAddress) {
+            return bytes32(0);
+        }
```

Comment:
im a little worried about footguns from this function being different than `KeyLib.hash`, could we just special case this inline in the tests?

Reply:
hmm what do you mean? this is a test function?

Reply:
ok I changed this so we have to explicitly use bytes32(0) in the tests

Reply:
I wonder if we should use not such a "default" value? Like it if should actually be some non zero value..

Reply:
i don't mind doing in a follow up we should track